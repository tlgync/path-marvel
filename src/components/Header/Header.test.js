import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Header } from '.';

test('Header Render SnapShot', () => {
  const onChange = () => {};
  const handleKeyPress = () => {};
  const handleSearch = () => {};
  const placeHolder = '';
  const value = '';

  const tree = renderer.create((
    <Router>
      <Header
        onChange={onChange}
        handleKeyPress={handleKeyPress}
        handleSearch={handleSearch}
        placeHolder={placeHolder}
        value={value}
      />
    </Router>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
