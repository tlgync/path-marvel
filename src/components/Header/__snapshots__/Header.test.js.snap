// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Render SnapShot 1`] = `
<div
  className="header"
>
  <div
    className="col-3"
  >
    <input
      onChange={[Function]}
      onKeyPress={[Function]}
      placeholder=""
      value=""
    />
    <span
      className="focus-bg"
    />
    <button
      onClick={[Function]}
      type="button"
    >
      Search
    </button>
  </div>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="logo"
      src="../img/logo.svg"
    />
  </a>
</div>
`;
